rustfmt.toml

```diff
@@ -0,0 +1,18 @@
+# Basic
+hard_tabs = true
+max_width = 100
+use_small_heuristics = "Max"
+# Imports
+imports_granularity = "Crate"
+reorder_imports = true
+# Consistency
+newline_style = "Unix"
+# Misc
+chain_width = 80
+spaces_around_ranges = false
+match_arm_leading_pipes = "Preserve"
+match_arm_blocks = false
+match_block_trailing_comma = true
+trailing_comma = "Vertical"
+trailing_semicolon = false
+use_field_init_shorthand = true
```

Comment:
These are all settings that substrate uses. For me, it's nice to use the same settings, as it feels familiar to the code-base I work most with. :)

I omitted two, which I did not deem helpful:

```
binop_separator = "Back" // see last commit to see the changes.
reorder_impl_items = false // prevents putting `type` and `const` as first entries in trait declarations.
```